ui/qr-code/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+plugins {
+    id("godtools.library-conventions")
+}
+
+android {
+    namespace = "org.cru.godtools.qrcode"
+
+    configureCompose(project)
+}
+
+dependencies {
+    implementation(libs.androidx.activity.compose)
+    implementation(platform(libs.androidx.compose.bom))
+    implementation(libs.androidx.compose.ui)
+    implementation(libs.androidx.compose.ui.graphics)
+    implementation(libs.androidx.compose.ui.tooling.preview)
```

Comment:
this dependency can be removed, it's already defined via `configureCompose` in build-config